model/operation.cc

```diff
@@ -142,24 +142,15 @@ namespace minsky
     bool notflipped=(fm>-90 && fm<90) || fm>270 || fm<-270;
     Rotate r(rotation()+(notflipped? 0: 180),0,0); // rotate into variable's frame of reference
     double z=zoomFactor();
-    float dx=xx-x(), dy=yy-y(), w=iWidth()*z, h=iHeight()*z; 
-    // make sure resize handles can be grabbed at corners of coupled integral variable. for feature 94.
-    if (const IntOp* i=dynamic_cast<const IntOp*>(this))
-      if (i->coupled())
-        {    
-          float dl=xx-i->x(), dr=xx-i->intVar->x(), wl=i->iWidth()*z, wr=i->intVar->iWidth()*z+i->intVarOffset;
-          if (((fabs(fabs(dl)-wl) < 0.5*portRadius*z) || (fabs(fabs(dr)-wr) < 0.5*portRadius*z)||
-               (fabs(hypot(dl,dy)-hypot(wl,h)) < 0.5*portRadius*z) || (fabs(hypot(dr,dy)-hypot(wr,h)) < 0.5*portRadius*z)) &&
-              fabs(fabs(dy)-h) < 0.5*portRadius*z 
-              )
-            return ClickType::onResize;		  
-        } 
-    if (fabs(fabs(dx)-w) < portRadius*z &&
-        fabs(fabs(dy)-h) < portRadius*z &&
-        fabs(hypot(dx,dy)-hypot(w,h)) < portRadius*z)
+    float dx=xx-x(), dy=yy-y();
+    // Ops, vars and switch icon only resize from bottom right corner. for ticket 1203 
```

Comment:
Reading ticket #1203, I don't think there is a requirement to preserve resize handles on all corners for Ravels, plots and sheets. Simplest might be to have a single resize handle on all canvas items.

Also, I think we have a problem with rotated items. Best if the resize behaviour was the same for all to fix that.